eslint.config.mjs

```diff
@@ -1,23 +1,28 @@
 // @ts-check
 
-const eslint = require('@eslint/js')
-const tseslint = require('typescript-eslint')
+import eslint from '@eslint/js'
+import tseslint from 'typescript-eslint'
+import stylisticTs from '@stylistic/eslint-plugin-ts'
 
-module.exports = tseslint.config(
+export default tseslint.config(
   {
     ignores: [
       '**/.keystone/',
+      '**/.next/',
       '**/dist/',
+      '**/__generated__/',
       '**/node_modules/',
       '**/syntax-error.js',
       '**/public/',
-      'examples/extend-graphql-schema-nexus/nexus-types.ts'
+      'examples/',
```

Comment:
I decided to completely ignore examples, it may be a pain to maintain. I can revert if needed.